Dictionaries/nld_OCRFixReplaceList.xml

```diff
@@ -23,6 +15,11 @@
   <EndLines />
   <WholeLines />
   <RegularExpressions>
+    <RegEx find="\blk(?=\p{Ll}{2})" replaceWith="Ik" />
     <RegEx find="\bln(?=\p{Ll}{2})" replaceWith="In" />
+    <RegEx find="\bls(?=\p{Ll}{2})" replaceWith="Is" />
+    <RegEx find="\beIk(aar|e|)" replaceWith="elk\1" />
+    <RegEx find="\blsol(atie|eerde|eert|eer|eren|ement)" replaceWith="Isol\1" />
+    <RegEx find="\bler(land|se|s|)\b" replaceWith="Ier\1" /> 
```

Comment:
The .NET regex engine understands both, but the former is undocumented. I suppose Microsoft wanted backward compatibility, yet, keep the option to drop it open.

Just, a minor optimisation:
```
<RegEx find="\beIk" replaceWith="elk" />
<RegEx find="\beIk(aar|e|)" replaceWith="elk$1" />
```
Both perform the  same transformation, but the former is more efficient (no capture).